_rules/meta-viewport-b4f0c3.md

```diff
@@ -0,0 +1,219 @@
+---
+id: b4f0c3
+name: meta viewport does not prevent zoom
+rule_type: atomic
+description: |
+  This rule checks that the `meta` element is not used to block the user agent ability to zoom. 
+accessibility_requirements:
+  wcag21:1.4.4: # Resize text (AA)
+    forConformance: true
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+input_aspects:
+  - DOM Tree
+authors:
+  - Jey Nandakumar
+  - Audrey Maniez
+---
+
+## Applicability
+
+The rule applies to each `<meta name="viewport">` element with a `content` attribute.
+
+## Expectation
+
+The `content` attribute of the test target: 
+- either does not have the property `user-scalable`, or has it set to yes; and
+- either does not have the property `maximum-scale`, or has it set to a value that allows significant zooming.
```

Comment:
How would you define "significant zooming"?

Reply:
@DagfinnRomen I have updated this to be something more meaningful.